docs/configuration-reference/components/prometheus-operator.md

```diff
@@ -33,6 +33,9 @@ component "prometheus-operator" {
 
   grafana {
     admin_password = "foobar"
+    secret_env     = {            # Declare variable to define this parameter and define it in `lokocfg.vars`.
```

Comment:
```suggestion
    secret_env     = {            # This might contain sensitive information, declare a variable and define this in `lokocfg.vars`.
```